src/components/Meanings.js

```diff
@@ -0,0 +1,70 @@
+import sourceIcon from '../assets/images/icon-new-window.svg';
+// import { v4 as uuid } from 'uuid';
+
+
+const Meanings = ({ searchedWord}) => {
+    const renderedMeanings = searchedWord.meanings.map((meaning, index) => {
+        console.log(meaning);
+
+        return (
+            <section className="displayMeaning" key={index} >
+
+                {/* PART OF SPEECH */}
+                <div className="displayPartOfSpeech">
+                    <h3 className="partOfSpeech">{meaning.partOfSpeech}</h3>
+                    <div className="displayLine"></div>
+                </div>
+
+                {/* DEFINITIONS */}
+                <div className="displayDefinition" >
+                    <h4>Meaning</h4>
+                    <ul>
+                        {
+                            meaning.definitions.map((def, index) => {
+                                return (
+                                    <li key={index}>
```

Comment:
I would advise using a unique id inside the map function because using the index of the array has issues with Re-ordering. you could use the above written v4 package to generate a unique id. you could use that this way-
 ``` JS
 <li key={v4()}>
 `